src/renderer/components/swap/Swap.tsx

```diff
@@ -591,20 +603,107 @@ export const Swap = ({
     [swapFeesRD, oPriceSwapOutFee, oSwapOutFee]
   )
 
+  // Affiliate fee
+  const affiliateFee: CryptoAmount = useMemo(
+    () =>
+      FP.pipe(
+        oQuote,
+        O.fold(
+          () => new CryptoAmount(baseAmount(0), AssetRuneNative), // default affiliate fee asset amount
+          (txDetails) => txDetails.txEstimate.totalFees.affiliateFee
+        )
+      ),
+    [oQuote]
+  )
+
+  // Price of swap OUT fee from oQuote // get poolPrice value and return
+  const oPriceAffiliatetFee: CryptoAmount = useMemo(() => {
+    const result = FP.pipe(
+      O.some(affiliateFee),
+      O.fold(
+        () => new CryptoAmount(baseAmount(0, targetAssetDecimal), targetAsset),
```

Comment:
Why target asset and not source asset?  It is taken off before the swap is done.